README.md

```diff
@@ -112,7 +112,40 @@ print(response)
 
 ## Examples
 
-You can find an example project in the [examples](https://github.com/iotaledger/iota.py/tree/master/examples) directory.
+We have a list of test cases in the [`examples` directory](https://github.com/iotaledger/iota.py/tree/master/examples) that you can use as a reference when developing apps with IOTA.
+
+Here's how you could send a zero-value transaction, using the library. For the guide, see the [documentation portal](https://docs.iota.org/docs/client-libraries/0.1/how-to-guides/python/send-your-first-bundle).
+
+```python
+# You don't need a seed to send zero-value transactions
+api = Iota('https://nodes.devnet.iota.org:443', testnet = True)
+
+# Define a message to send.
+# This message must include only ASCII characters.
```

Comment:
💭 PyOTA does support any Unicode code points; it encodes the value using UTF-8 before translating into trytes.

Reply:
I thought we weren't recommending UTF-8 characters at the moment because this method isn't standardized across the libraries and may change.